pubspec.yaml

```diff
@@ -25,15 +25,15 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  permission_handler: ^10.0.0
+  permission_handler: ^11.0.0
   shared_preferences: ^2.0.15
   bluedot_point_sdk:
     # When depending on this package from a real application you should use:
     #   bluedot_point_sdk: ^x.y.z (^1.0.1)
     # See https://dart.dev/tools/pub/dependencies#version-constraints
     # The example app is bundled with the plugin so we use a path dependency on
     # the parent directory to use the current plugin's version.
-    path: ../
+    path: ../PointSDK-Flutter-Plugin/
```

Comment:
[Q]Should we replace this local path with bluedot_point_sdk: ^2.0.0 so a client can use the min app without fixing the path

Reply:
Yes, after the release of plugin. I will update it to version ^2.0.0. I am not merging this PR until then.

Reply:
It's good to go to `dev/2.0.0` branch instead of`main`.